app/components/AddressChooserModalComponent.tsx

```diff
@@ -2,14 +2,17 @@ import TextField from "material-ui/TextField";
 import * as React from "react";
 import { Button, Modal } from "react-bootstrap";
 
-import { IAddresses } from "../containers/AddressChooserModalContainer";
+import { IDictionary } from "../types";
 import * as style from "./AddressChooserModalComponent.scss";
 import { LoadingIndicator } from "./LoadingIndicator";
 
 interface IAddressChooserModalComponent {
   loading: boolean;
   derivationPath: string;
-  addresses: IAddresses;
+  addresses: IDictionary<{
```

Comment:
could you create type for and put it to `types.ts`?
```
{
    address?: string;
    ETH?: number;
}```

Also, why these fields optional?